libs/common/src/sql-db-entities/utilisation-report/utilisation-report.types.ts

```diff
@@ -0,0 +1,7 @@
+import { ReportPeriodPartialEntity } from '../partial-entities';
+import { DbRequestSourceParam } from '../helpers';
+
+export type CreateNotReceivedUtilisationReportEntity = DbRequestSourceParam & {
```

Comment:
How about a name like `CreateNotReceivedUtilisationReportParams`?

Reply:
Agreed, that's clearer 👍